app/frontend/schools/coaches/types/school_index/CoachesSchoolIndex__Coach.res

```diff
@@ -72,10 +72,6 @@ let make = (
 }
 
 let updateList = (coaches, coach) => {
-  let oldList = coaches |> List.filter(t => t.id !== coach.id)
-  oldList
-  |> List.rev
-  |> List.append(list{coach})
-  |> List.rev
-  |> List.sort((x, y) => int_of_string(x.id) - int_of_string(y.id))
+  let oldList = coaches->Js.Array2.filter(t => t.id !== coach.id)
+  oldList->Js.Array2.concat([coach])
```

Comment:
All of this arises from not choosing to reload the page. We should switch to reload on create and update. This will let us paginate the list properly. 

1. Let's reload the page always. 
2. Implement pagination for the list; Like how we did for the courses export

Reply:
Not now 😄